src/components/Lesson.vue

```diff
@@ -258,11 +262,27 @@ export default {
         return new IPFS({repo: Math.random().toString()})
       }
     },
+    clearCache: function () {
+      let cacheKey = this.$attrs.exercise + this.$attrs.lessonTitle
+      delete localStorage[cacheKey]
+      this.cachedCode = false
+      if (this.code !== this.$attrs.code) {
+        this.code = this.$attrs.code || defaultCode
+        this.editor.setValue(this.code)
+      }
+    },
     onMounted: function (editor) {
       this.editor = editor
     },
     onCodeChange: function (editor) {
-      // console.log(editor.getValue())
+      if (editor.getValue() === this.$attrs.code) {
```

Comment:
`this.$attrs.code` probably needs to be `(this.$attrs.code || defaultCode)` to handle the case where the code property wasn't set by the lesson.

Reply:
@mikeal Since I'm evaluating, not setting, then what you're proposing would mean that it evaluates to true if the code in the editor matches either thing, right? If `this.$attrs.code` exists, then I only want it to evaluate to true if it matches that, not `defaultCode`.   If `this.$attrs.code` exists but the editor matches `defaultCode` instead, I need it to evaluate to false. Do I need to something more like this? 

`if ((this.$attrs.code && editor.getValue() === this.$attrs.code) || (!this.$attrs.code && editor.getValue() === defaultCode))`

Reply:
The simplest way is to just do an inline OR operator in the assignment:

```javascript
if (editor.getValue() === (this.$attrs.code || defaultCode) )
```

Reply:
Ah, okay, so putting the parens around those options will force it to match the first if the first exists. Thanks!